packages/cozy-pouch-link/src/remote.js

```diff
@@ -62,7 +62,7 @@ export const fetchRemoteLastSequence = async baseUrl => {
   const remoteUrl = new URL(`${baseUrl}/_changes`)
   const res = await fetchRemoteInstance(remoteUrl, {
     limit: 1,
-    descending: true
+    ...(baseUrl.includes('sharings/drives') ? {} : { descending: true }) // shared drives' _change route does not support descending. It look like it has no impact on the expected result
```

Comment:
I don't understand how it can works: this is done to retrieve the lastSeq from the lastest change. So either you retrieve the wrong lastSeq, either the descending order is true by default for shared drives (which is normally false by default). @nono can you help us? This might require to add the descending order on the stack side (or maybe there is a simpler way to retrieve the lastSeq?)

Reply:
I see, since I don't do try to do the replication with PouchDb replication anymore, this change is not needed anymore. I remove it

Reply:
> if there are some documents added during the replication

if we retrieve `lastSeq=10` and new docs are added afterwards, making `lastSeq = 12` we will get them next time we replicate as we'll ask docs since 10. But it might be problematic if it fails indeed.  

> You should use the lastSeq from last successful call of the replication for that.

Yes, it's probably better. @doubleface we can plug it [here](https://github.com/cozy/cozy-client/blob/master/packages/cozy-pouch-link/src/startReplication.js#L106) and listen for `replication.complete` to get the lastSeq. 
See the example given in https://pouchdb.com/api.html#replication

Reply:
And I use the lastSeq from the previous request for this.